src/com/sun/ts/tests/servlet/api/jakarta_servlet/filter/DoFilterTestServlet.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ * *
+ * * The Apache Software License, Version 1.1
+ * *
+ * * Copyright (c) YYYY Oracle and/or its affiliates. All rights reserved.
```

Comment:
Don't know if this is a bug in the copyright plugin. But this was 2018 instead of YYYY in previous version of the file. Maybe the folder name change of the file is causing this? Same for more files.

Reply:
looking at it (manually 😩)

Reply:
this one is interesting https://github.com/eclipse-ee4j/jakartaee-tck/blob/a4a509c65325cecdc9c4a9469678ed3239137af7/src/com/sun/ts/tests/servlet/api/javax_servlet/genericfilter/InitFilterConfigTestServlet.java#L6

Reply:
so it is YYYY in master... from this commit https://github.com/eclipse-ee4j/jakartaee-tck/commit/a0fb64cf8f81ef56318701cc1e11c8b14e844ce8 cc @cesarhernandezgt

Reply:
@alwin-joseph I have fixed those manually (comparing each files to be sure having correct range)